uvls/src/webview.rs

```diff
@@ -823,7 +823,7 @@ pub async fn ui_main(
 //HTTP server for the configuration interface
 pub async fn web_handler(pipeline: AsyncPipeline, port: u16) {
     info!("Starting web handler");
-    let addr: std::net::SocketAddr = ([127, 0, 0, 1], port).into();
+    let addr: std::net::SocketAddr = ([0, 0, 0, 0], port).into();
```

Comment:
Why is that needed?

Reply:
The address will be used later to bind the WebSocket to. Using localhost made it impossible to call from a remote location. However, as the Playground will drop Configuration as a feature, this change is no longer needed and therefore reverted.